### ISSUE TITLE
Inconsistent or missing focus indicators

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4982]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.xx
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
Use keyboard to navigate around the RStudio IDE (where possible, not possible in many parts of the UI currently; separate issue not tracked here).

### Describe the problem in detail
Currently, RStudio IDE doesn't indicate focus at all in many cases (e.g. buttons), indicates it in insufficiently visible fashion (checkboxes), shows it differently across browsers (user-agent styling of some text boxes), or shows it in an inconsistent/ugly manner (toolbar buttons, tab control such as the vertical selection control used in preferences dialogs).

### Describe the behavior you expected
To meet basic accessibility standards, focus must be visually indicated when navigating with the keyboard (e.g. tab key, arrows).

### More details
Much of the IDE is not keyboard accessible at all (as of this writing), so this will be a work in progress during 1.3 and visual quirks will pop up as things are made keyboard focusable for the first time.

Some potential angles:

1. Use standard browser focus behavior and styling
2. Use standard browser focus behavior, but restyle for consistency across browsers
3. User-option to turn focus indicators on/off (in accessibility preferences)
4. Consider using the `:focus-visible` pseuoselector (polyfilled since browsers don't support it yet). https://drafts.csswg.org/selectors-4/#the-focus-visible-pseudo

**Comment by gtritchie** [comment]
Marking as closed. We should open individual issues at this point for areas of the UI that still have this problem.
